### PR TITLE
YD-142 embed message after action

### PR DESCRIPTION
**Pull request by aristotelos**

### CODE REVIEW

File: core/src/main/java/nu/yona/server/messaging/service/MessageService.java [hunk 1]
```diff
@@ -209,4 +209,14 @@ public void removeMessagesFromUser(MessageDestinationDTO destination, UUID sentB
 		destinationEntity.removeMessagesFromUser(sentByUserAnonymizedID);
 		MessageDestination.getRepository().save(destinationEntity);
 	}
+
+	public static boolean getIsDirectMessage(MessageDTO message)
```
Comment (Bert-R) [inline core/src/main/java/nu/yona/server/messaging/service/MessageService.java]: Please follow the common naming pattern: ``isDirectMessage(...)``.

File: core/src/main/java/nu/yona/server/messaging/service/MessageService.java [hunk 2]
```diff
@@ -209,4 +209,14 @@ public void removeMessagesFromUser(MessageDestinationDTO destination, UUID sentB
...
+	{
+		if (message instanceof BuddyConnectRequestMessageDTO)
```
Comment (Bert-R) [inline core/src/main/java/nu/yona/server/messaging/service/MessageService.java]: This approach is a little tricky. Over time, we might add more types of direct messages or, even worse, send some type of message through both message boxes. Let's for now stick with it, as it makes the overall code flow mor readable.

File: core/src/main/java/nu/yona/server/messaging/service/DiscloseRequestMessageDTO.java [hunk 3]
```diff
@@ -122,33 +121,39 @@ public MessageActionDTO handleAction(UserDTO actingUser, Message messageEntity,
...
+		private MessageActionDTO handleAction_Reject(UserDTO actingUser, DiscloseRequestMessage discloseRequestMessageEntity,
 				MessageActionDTO payload)
 		{
-			return updateResult(rejectingUser, discloseRequestMessageEntity, GoalConflictMessage.Status.DISCLOSE_REJECTED,
+			return updateResult(actingUser, discloseRequestMessageEntity, GoalConflictMessage.Status.DISCLOSE_REJECTED,
 					payload.getProperty("message"));
 		}
 
-		private MessageActionDTO updateResult(UserDTO respondingUser, DiscloseRequestMessage discloseRequestMessageEntity,
+		private MessageActionDTO updateResult(UserDTO actingUser, DiscloseRequestMessage discloseRequestMessageEntity,
```
Comment (Bert-R) [inline core/src/main/java/nu/yona/server/messaging/service/DiscloseRequestMessageDTO.java]: I know the method name is not changed, but still: This method name is not very logical to me. What about ``buildReturnValue``?

**Comment by Bert-R** [comment]
Looks good. I added a few minor comments.

File: core/src/main/java/nu/yona/server/messaging/service/DiscloseRequestMessageDTO.java (hunk 3 above)
Comment (aristotelos) [inline core/src/main/java/nu/yona/server/messaging/service/DiscloseRequestMessageDTO.java]: Renamed to more speaking `updateGoalConflictMessageStatus`

File: core/src/main/java/nu/yona/server/messaging/service/MessageService.java (hunk 1 above)
Comment (aristotelos) [inline core/src/main/java/nu/yona/server/messaging/service/MessageService.java]: Indeed, done

**Comment by aristotelos** [comment]
Reworked

**Comment by Bert-R** [comment]
But not committed or not pushed...

**Comment by aristotelos** [comment]
Oops, indeed not pushed correctly, done now

**Comment by Bert-R** [comment]
Looks good. I'll merge it after the build of the previous one is completed (a second build while one is active already might break the test run, as that replaces the docker images while the tests are active).

**Pull request merged**